<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiApplication1Class</class>
 <widget class="QMainWindow" name="QtGuiApplication1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiApplication1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="hostEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Host</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passEdit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Pass</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portEdit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="conBtn">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>1001</width>
      <height>251</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>파일(F)</string>
    </property>
    <addaction name="SiteManage"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>보기(V)</string>
    </property>
    <addaction name="actionDebug_Mode"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_V"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="SiteManage"/>
   <addaction name="separator"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionCancel"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionReconnet"/>
  </widget>
  <action name="SiteManage">
   <property name="icon">
    <iconset resource="QtGuiApplication1.qrc">
     <normaloff>:/QtGuiApplication1/SiteManage</normaloff>:/QtGuiApplication1/SiteManage</iconset>
   </property>
   <property name="text">
    <string>SiteManage</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDebug_Mode">
   <property name="text">
    <string>Debug Mode</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QtGuiApplication1.qrc">
     <normaloff>:/QtGuiApplication1/I:/Skins For GUI Programming/Icon/refresh.png</normaloff>:/QtGuiApplication1/I:/Skins For GUI Programming/Icon/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QtGuiApplication1.qrc">
     <normaloff>:/QtGuiApplication1/I:/Skins For GUI Programming/Icon/cancel.png</normaloff>:/QtGuiApplication1/I:/Skins For GUI Programming/Icon/cancel.png</iconset>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </action>
  <action name="actionReconnet">
   <property name="icon">
    <iconset resource="QtGuiApplication1.qrc">
     <normaloff>:/QtGuiApplication1/I:/Skins For GUI Programming/Icon/reconnect.png</normaloff>:/QtGuiApplication1/I:/Skins For GUI Programming/Icon/reconnect.png</iconset>
   </property>
   <property name="text">
    <string>reconnect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="QtGuiApplication1.qrc">
     <normaloff>:/QtGuiApplication1/I:/Skins For GUI Programming/Icon/disconnect.png</normaloff>:/QtGuiApplication1/I:/Skins For GUI Programming/Icon/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>disconnect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGuiApplication1.qrc"/>
 </resources>
 <connections/>
</ui>
